gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/impl/sql/GenericStatement.java

```diff
@@ -151,7 +151,7 @@ Licensed to the Apache Software Foundation (ASF) under one or more
             Pattern.compile(".*PUT\\s+INTO\\s+(TABLE)?.*SELECT\\s+.*",
                 Pattern.CASE_INSENSITIVE);
         private static final Pattern EXECUTION_ENGINE_STORE_HINT =
-            Pattern.compile(".*EXECUTIONENGINE(\\s+)?+=(\\s+)?+STORE\\s+.*",
+            Pattern.compile(".*EXECUTIONENGINE(\\s+)?+=(\\s+)?+STORE.*[\\r\\n].*",
```

Comment:
It doesn't match if "\n" is in the string. 
e.g. The following query doesn't match:
select * from tabOne --GEMFIREXD-PROPERTIES executionEngine=Store, index=indexThree \n where id2 = 111
but matches if I remove the '\n'